libs/components/forms/src/lib/modules/checkbox/checkbox.component.html

```diff
@@ -51,5 +51,20 @@
       />
     </ng-template>
   </span>
-  <ng-content select="sky-checkbox-label" />
+  <ng-container *ngIf="labelText; else labelElement">
+    <sky-checkbox-label>{{ labelText }}</sky-checkbox-label>
+  </ng-container>
+  <ng-template #labelElement>
+    <ng-content select="sky-checkbox-label" />
+  </ng-template>
 </label>
+<div *ngIf="labelText && ngControl?.errors">
+  <sky-form-errors
+    [id]="errorId"
+    [errors]="ngControl?.errors"
```

Comment:
wouldn't it be better to send this input for `sky-form-error` only the value for `this.required` to be more explicit about which error we're sending

Reply:
This section also projects any custom validation error messages the consumer provides via the ng-content. We're not sure what their errors are so we can't be more specific than that. I'm not entirely sure _what_ custom validation someone might need to do on a checkbox, but I'm sure there is something 😂

Reply:
right but `sky-form-error` would consume and display that from an `ng-content` inside of it and not from this input afaik